Dockerfile

```diff
@@ -64,21 +76,31 @@ RUN apt-get update \
     zlib1g \
   && rm -rf /var/lib/apt/lists/*
 
-# Install protoc 3.8.0.
-RUN mkdir -p /usr/src/protoc/ \
-  && curl --location https://github.com/google/protobuf/releases/download/v3.8.0/protoc-3.8.0-linux-x86_64.zip > /usr/src/protoc/protoc.zip \
-  && cd /usr/src/protoc/ \
+# Install all versions of protoc needed by all the languages.
+
+# Install protoc ${PROTOBUF_PYTHON_VERSION} for Python and other languages that use this version.
```

Comment:
Using Python for other languages seems a bit less straightforward.

Would it be possible first figure out all the versions of protoc needed and then download and link them?

Reply:
Ok! Moved the logic into a bash script `install_protoc.sh`.